src/bluesky/callbacks/tiled_writer.py

```diff
@@ -75,12 +78,16 @@ def _ensure_resource_backcompat(self, doc: StreamResource) -> StreamResource:
 
         doc = copy.copy(doc)
         if ("mimetype" not in doc.keys()) and ("spec" not in doc.keys()):
-            raise RuntimeError("StreamResource document is missing a mimetype or spec")
+            raise RuntimeError("StreamResource document is missing a 'mimetype' or 'spec'")
```

Comment:
Since this fails if true the else below is superfluous, we can remove it and unindent the contents.